include/cantera/kinetics/Falloff.h

```diff
@@ -22,24 +23,51 @@ class AnyMap;
 
 /**
  * Base class for falloff function calculators. Each instance of a subclass of
- * Falloff computes one falloff function. This base class implements the
- * trivial falloff function F = 1.0.
- *
- * @ingroup falloffGroup
+ * Falloff computes one falloff function.
  */
-class Falloff
+class FalloffRate : public ReactionRate
```

Comment:
The class docstring here needs to be updated, both to say `FalloffRate` instead of `Falloff`, and also to address the fact that this fully implements the calculation of a falloff reaction rate, not just the "falloff function"